### ISSUE TITLE
Expand falling behavior

### ISSUE THREAD

**Issue by mciul** [i7/kerkerkruip#398]
Having the tentacle able to throw opponents begs the question: What happens if they're thrown in a place where they can fall? Right now, the falling system is full of special cases and it would be hard to implement this behavior without adding more special cases. I'd like to expand the falling system so effects like this would be easier to implement.

Once that is in place, it might help to implement the free-fall room idea I had, and maybe there could be a new room with shifting rubble on the floor, where people fall but don't leave the room. Perhaps they'd take an initiative penalty or have to "regroup" like the swarm of daggers... Hey, I like this idea! Perhaps items in the room could appear and disappear as the rubble shifts! You could search the rubble but run the risk that some of it would collapse on you...

**Comment by VictorGijsbers** [comment]
Good idea. I'll implement the tentacle idea right now (well, soon), maybe we should schedule the rest for Kerkerkruip 11?

**Comment by VictorGijsbers** [comment]
Wait, it's probably quite hard to implement this for the tentacle, because rooms don't have a property like "falling risk", isn't it? Yeah, I thought so. Hm... shall we still special case it for the staircase and the bridge of doom? It would make the tentacle on the bridge of doom VERY HARD! But that might be fun. :-)

**Comment by mciul** [comment]
If you don't mind handling the special cases, I think it would be nice to have that working in 10. But I think it would be fine if you'd rather not. Rescheduling the rest for 11 sounds good!
